src/openMarket/infrastructure/order/LocalStorageOrderRepository.js

```diff
@@ -13,7 +13,15 @@ export default class LocalStorageOrderRepository extends OrderRepository {
   }
 
   findById({id}) {
-    return null;
+    return RxLocalStorage.loadLocalStorage({ localStorageKey: this._localStorageKey })
```

Comment:
So this should be injected or this class renamed to RxLocalStorageOrderRepository

Reply:
you are right, this should be injected, I can't remember why the first implementation of RxLocalStorage is static, anyway I need to apply a big refactor in all repos based on RxLocalStorage
:+1: